dashboard/app/controllers/courses_controller.rb

```diff
@@ -34,6 +34,7 @@ def index
     @is_english = request.language == 'en'
     @is_signed_out = current_user.nil?
     @force_race_interstitial = params[:forceRaceInterstitial]
+    @courses_announcement = Announcements.get_localized_announcement_for_page("/courses")
```

Comment:
Just for my understanding, where is this specific banner getting assigned to the /courses page so this returns the correct one?

Reply:
the `get_localized_announcement_for_page` method makes use of the `announcements.json` which is where the correct banner is assigned for /courses page